### PR TITLE
Feature/network api

### PR DESCRIPTION
**Pull request by Elestrias**

### CODE REVIEW

**Review by turuslan — APPROVED** [review]

File: core/api/network/setup_net.hpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <libp2p/host/host.hpp>
+#include "api/network/network_api.hpp"
+
+namespace fc::api {
+  using libp2p::Host;
```
Comment (ortyomka) [inline core/api/network/setup_net.hpp:12]: Not used

File: core/api/storage_miner/storage_api.hpp [hunk 2]
```diff
@@ -53,6 +55,8 @@ namespace fc::api {
   using sector_storage::stores::SectorIndex;
   using StorageInfo_ = sector_storage::stores::StorageInfo;
 
+  const static common::Logger kStorageApiLogger = common::createLogger("storage API");
```
Comment (ortyomka) [inline core/api/storage_miner/storage_api.hpp:58]:
```suggestion
  const static common::Logger kStorageApiLogger = common::createLogger("Storage API");
```

File: core/node/blocksync_request.cpp [hunk 3]
```diff
@@ -434,11 +434,14 @@ namespace fc::sync::blocksync {
...
-                       response.message,
-                       response.chain.size());
+          log()->debug(
+              "got response from {}: status={}, msg=({}), size={}, "
+              "block_height={}",
+              result_->from->toBase58(),
+              statusToString(response.status),
+              response.message,
+              response.chain.size(),
+              response.chain[0].blocks[0].height);
```
[the remark this answers is not in the document]
Comment (ortyomka) [reply core/node/blocksync_request.cpp:444]: Still in the code

File: core/api/network/setup_net.hpp [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+  inline void fillNetApi(const std::shared_ptr<NetworkApi> &api,
+                         const PeerInfo &api_peer_info,
+                         const std::shared_ptr<libp2p::Host> &host,
```
Comment (Alexey-N-Chernyshov) [inline core/api/network/setup_net.hpp:16]: Either use Host or remove `using libp2p::Host;` 
```suggestion
                         const std::shared_ptr<Host> &host,
```

File: core/miner/main/main.cpp [hunk 5]
```diff
@@ -497,7 +497,7 @@ namespace fc {
                                     storage_provider,
                                     retrieval_provider);
 
-    api::fillAuthApi(mapi, api_secret, log());
+    api::fillAuthApi(mapi, api_secret, api::kStorageApiLogger);
```
Comment (ortyomka) [inline core/miner/main/main.cpp:500]: Should `fillNetApi` be here?

**Review by ortyomka — APPROVED** [review]

**Review by Alexey-N-Chernyshov — APPROVED** [review]

**Pull request merged**